### DISCUSSION TITLE
Installing `x86_64` based apps/tools on `arm64e` homebrew

### DISCUSSION THREAD

**Discussion by kennedy** [Homebrew/discussions#3021] [Everyday usage] [answered]
Hello,
I am running an M1 computer but there are some `x86_64` based CLI apps I am still able to run via Rosetta, such as `terraform@0.13.7`. I am able to manually place the `x86_64` binary in my `PATH` and launch it without any issues.

How can I install `x86_64` apps on `arm64e` homebrew?

This will enable me to continue using `brew pin` and upgrade when my organization moves to an the `arm64e` variant of the tools.

**Comment by gromgit** [comment, marked as answer]
> How can I install `x86_64` apps on `arm64e` homebrew?

You shouldn't. Instead, you want to install an Intel Homebrew instance under `/usr/local`:
```
arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
then `/usr/local/bin/brew install ...` to add the (presumably few) formulae for which you need Intel binaries.

**Reply by kennedy** [reply]
Thanks for getting back to me so quickly.  

I was just looking at the [3.0.0 release post](https://brew.sh/2021/02/05/homebrew-3.0.0/). Seems like reinstalling a `x86_64` brew is the official stance on the subject. I suppose it would not be too cumbersome to maintain two brew management flows.
